dev/docker-compose.dev.carl.yaml

```diff
@@ -0,0 +1,11 @@
+# docker-compose development override for carl service
+---
+version: "3.4"
+services:
+  hydrant:
```

Comment:
```suggestion
  carl:
```

Reply:
Nice catch, thanks! Too bad it's obvious where I lifted that from now ;P